EpsilonCCS/Src/CanParsers/MotorCanParser.c

```diff
@@ -140,7 +140,7 @@ void parseMotorCanMessage(uint32_t stdId, uint8_t* data)
 
         case M1_VELOCITY_CAN_ID:
             // motorVelocity = arrayToFloat(&data[0]);
```

Comment:
This comment must have been here for some random reason. It should be safe to remove it.